lib/setup.py

```diff
@@ -49,6 +49,8 @@
     "tenacity>=8.1.0, <10",
     "toml>=0.10.1, <2",
     "typing-extensions>=4.3.0, <5",
+    "py-redis>=1.1.1",
```

Comment:
Why don't we use the [official redis python package](https://pypi.org/project/redis/)?
[py-redis](https://pypi.org/project/py-redis/) doesn't seem to be maintained.